integration_test/exqlite/ecto_sql/migration.exs

```diff
@@ -461,7 +461,6 @@ defmodule Ecto.Integration.MigrationTest do
     assert :ok == down(PoolRepo, num, OnDeleteMigration, log: false)
   end
 
-  @tag :composite_foreign_key
```

Comment:
removed since it was an addition from us that is no longer needed (since we support composite_fk now)